src/formdata.js

```diff
@@ -0,0 +1,66 @@
+import 'core-js/stable';
+import 'regenerator-runtime/runtime';
+import renderHome from './home.js';
+
+renderHome();
+const form = document.querySelector('#form-id');
+const refresh = document.querySelector('#refresh-score-btn');
+const table = document.querySelector('table');
+
+const getForm = (e) => {
+  e.preventDefault();
+  const name = document.querySelector('#name-field');
+  const score = document.querySelector('#score-field');
+
+  // fetch to POST the data:
+  fetch(
+    'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/TAndiRIq5l6QgrYO74Z7/scores',
+    {
+      method: 'POST',
+      body: JSON.stringify({
+        user: name.value,
+        score: score.value,
+      }),
+      headers: {
+        'Content-Type': 'application/json; charset=UTF-8',
+      },
+    },
+  )
+    .then((res) => res.json())
+    .then((data) => {
+      const message = document.querySelector('#message');
+      message.innerHTML = data.result;
+      message.style.backgroundColor = 'green';
+    });
```

Comment:
- [ ] <b>OPTIONAL:</b> This wasn't mentioned in the previous review, but as requested in project requirements, you have to use `async/await` in case of any API request in this project. Kindly do that to comply with the project specifications.